infra/log-analytics-cluster.tf

```
@@ -0,0 +1,34 @@
+## Resource Group da Rimuovere e sostituire con quello creato su Azure ##
+resource "azurerm_resource_group" "log_analytics_rg" {
+  name     = "${local.project}-log-analytics-rg"
+  location = "Italy North"
+}
+
+resource "azapi_resource" "log_analytics_cluster" {
+  type      = "Microsoft.OperationalInsights/clusters@2023-09-01"
+  name      = "${local.project}-log-analytics-cluster"
```

log analytics is not present in azure doc abbreviations but we can append `cluster`

https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#management-and-governance

```suggestion
  name      = "${local.project}-log-cluster"
```